src/Bridge/Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -184,37 +219,38 @@ protected function filterProperty(string $property, $value, QueryBuilder $queryB
      *
      * @throws InvalidArgumentException If strategy does not exist
      */
-    protected function addWhereByStrategy(string $strategy, QueryBuilder $queryBuilder, QueryNameGeneratorInterface $queryNameGenerator, string $alias, string $field, $value, bool $caseSensitive)
+    protected function addWhereByStrategy(string $strategy, QueryBuilder $queryBuilder, QueryNameGeneratorInterface $queryNameGenerator, string $alias, string $field, $value, bool $caseSensitive, ClassMetadata $metadata)
```

Comment:
This is a BC break. You should use `func_get_arg` and add a deprecated message if the parameter is not given.

Reply:
it's a protected function that has only one call throughout the whole code in the same file. 
Are you thinking about people extending the class ?

Reply:
Yes, some users are extending this class since it's not final (yet). See also: https://symfony.com/doc/current/contributing/code/bc.html.